docs/docs/styling.md

```diff
@@ -146,6 +146,29 @@ Navigation.mergeOptions(this.props.componentId, {
       enabled: true
     }
   },
+  fab: {
+    id: 'fab',  // required
+    backgroundColor: 'green',
+    clickColor: 'blue',
+    rippleColor: 'yellow',
+    visible: true,
+    icon: require('add.png'),
```

Comment:
This has no effect when you have an actions, or i missed something to make this work?